src/libMessage/ZilliqaMessage.proto

```diff
@@ -240,21 +240,27 @@ message ProtoDiagnosticData
 // Primitives
 // ============================================================================
 
+message ProtoAccountBase
+{
+    required uint32 version    = 1;
+    required ByteArray balance = 2;
+    required uint64 nonce      = 3;
+    optional bytes codehash    = 4;
+    optional bytes storageroot = 5;
+}
+
 message ProtoAccount
 {
     message StorageData
     {
         required bytes keyhash       = 1; // Added in: v1.0, Deprecated in: N/A
         required string data         = 2; // Added in: v1.0, Deprecated in: N/A
     }
-    optional uint32 version        = 1;  // Added in: v1.0, Deprecated in: N/A
-    optional bool numbersign       = 2;  // For account delta (false=neg, true=pos) // Added in: v1.0, Deprecated in: N/A
-    required ByteArray balance     = 3;  // Added in: v1.0, Deprecated in: N/A
-    required uint64 nonce          = 4;  // Added in: v1.0, Deprecated in: N/A
-    optional bytes storageroot     = 5;  // Added in: v1.0, Deprecated in: N/A
-    optional bytes codehash        = 6;  // Added in: v1.0, Deprecated in: N/A
-    optional bytes code            = 7;  // Added in: v1.0, Deprecated in: N/A
-    repeated StorageData storage   = 8; // Added in: v1.0, Deprecated in: N/A
+    required ProtoAccountBase base = 1; 
```

Comment:
For all database-related message fields, I recommend to use optional. The enforcement of fields like base, version, balance, etc. can be done in code side. If done in protobuf storage side, it will be difficult to maintain.